test/FixCode/fixits-var-let-parameter.swift

```diff
@@ -0,0 +1,7 @@
+// RUN: not %swift -emit-sil -target %target-triple %s -emit-fixits-path %t.remap -diagnostics-editor-mode
+// RUN: c-arcmt-test %t.remap | arcmt-test -verify-transformed-files %s.result
+
+func foo(var arg: Int) {}
+func bar(let arg: Int) {}
+
+func baz(var arg: Int, var arg2: Int) {}
```

Comment:
Going through a full transformation is a bit heavy.  A simple verifier test that checks you nuked whitespace should suffice

Reply:
We have a fix-it syntax in double braces which could be used to make sure that it removes enough whitespace.